jax/numpy/lax_numpy.py

```diff
@@ -3320,6 +3319,24 @@ def unique(ar, return_index=False, return_inverse=False,
   raise NotImplementedError(
         "np.unique is not implemented for the axis argument")
 
+@_wraps(np.setxor1d)
+def setxor1d(ar1, ar2, assume_unique=False):
+  if not assume_unique:
+    ar1 = unique(ar1)
+    ar2 = unique(ar2)
+  else:
+    ar1 = asarray(ar1)
+    ar2 = asarray(ar2)
```

Comment:
We should not call `asarray`; valid inputs will be properly handled by `concatenate`.